app/models/user.rb

```diff
@@ -1,2 +1,11 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
+  private
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] NIce job on making this method that returns recents_post of the User, however this method (`recent_posts`) should not be private because it will be accessible outside of the User model.  kindly remove it in private methods.